silkworm/capi/silkworm.h

```diff
@@ -98,7 +98,23 @@ struct SilkwormChainSnapshot {
 #define SILKWORM_PATH_SIZE 260
 #define SILKWORM_GIT_VERSION_SIZE 32
 
+//! Silkworm library logging level
+enum SilkwormLogLevel : uint8_t {
```

Comment:
Is it the right place for the log level? I'd move it to capi/common or even to the top level.

Reply:
I understand your doubt, but it needs to stay here because the rule is:

**C API client code must include just "silkworm.h"**